server/rate-limiter.js

```diff
@@ -24,7 +24,7 @@ class KumaRateLimiter {
      */
     async pass(callback, num = 1) {
         const remainingRequests = await this.removeTokens(num);
-        log.info("rate-limit", "remaining requests: " + remainingRequests);
+        log.debug("rate-limit", "remaining requests: " + remainingRequests);
```

Comment:
Sorry, but this does not fix the underlying issue.

This being an `Info` level log is intentional and should not be changed.
What instead is the problem is that this log message should only fire if `remainingRequests` is significantly below `tokensPerInterval`

Reply:
Ok soryy,
then I misunderstood this comment: https://github.com/louislam/uptime-kuma/issues/5122#issuecomment-2488006140